src/components/NavigationBar/test/NavigationBar.test.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from 'react';
+import { mergeConfig } from '@edx/frontend-platform';
+import {
+  render, screen, cleanup, initializeMockApp, fireEvent,
+} from '../../../setupTest';
+import NavigationBar from '..';
+
+const mockHistoryPush = jest.fn();
+global.ResizeObserver = require('resize-observer-polyfill');
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useHistory: () => ({
+    push: mockHistoryPush,
+  }),
+}));
+
+describe('navigation-bar', () => {
+  beforeAll(async () => {
+    await initializeMockApp();
+  });
+  beforeEach(() => {
+    mergeConfig({ ENABLE_VERIFIABLE_CREDENTIALS: 'true' });
+    return jest.resetModules;
+  });
+  afterEach(cleanup);
+
+  it('not renders the component with disabled the Verifiable Credentials functionality', () => {
```

Comment:
nit: to word this test better, I think it can say "does not render the component with Verifiable Credentials functionality when flag is disabled"